ejector/ejector_config.go

```diff
@@ -70,6 +71,52 @@ type EjectorConfig struct {
 	EjectionFinalizationPeriod time.Duration
 }
 
+var _ config.VerifiableConfig = (*EjectorSecretConfig)(nil)
+
+// Configuration for secrets used by the ejector.
+type EjectorSecretConfig struct {
+	// The Ethereum RPC URL(s) to use for connecting to the blockchain.
+	EthRpcUrls []string `docs:"required"`
+
+	// The private key to use for signing ejection transactions.
+	PrivateKey string `docs:"required"`
+}
+
+// Create a new root ejector config with default values.
+func NewRootEjectorConfig() *RootEjectorConfig {
+	return &RootEjectorConfig{
+		Config: DefaultEjectorConfig(),
+		Secret: &EjectorSecretConfig{},
+	}
+}
+
+func (e *RootEjectorConfig) GetEnvVarPrefix() string {
+	return "EJECTOR"
+}
+
+func (e *RootEjectorConfig) GetName() string {
+	return "Ejector"
+}
+
+func (e *RootEjectorConfig) GetPackagePaths() []string {
+	return []string{
+		"github.com/Layr-Labs/eigenda/ejector",
+	}
+}
+
+// Verify implements config.DocumentedConfig.
```

Comment:
This doc is a bit odd. It's technically correct, since `VerifiableConfig` is nested inside `DocumentedConfig`, but seems a bit misleading. Also, is there a reason to explicitly say this for `Verify`, but not for the other interface methods?